sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/StoreResult.java

```diff
@@ -57,7 +58,8 @@ public StoreResult(
             long globalCommittedLSN,
             int numberOfReadRegions,
             long itemLSN,
-            ISessionToken sessionToken) {
+            ISessionToken sessionToken,
+            Double backendLatencyInMs) {
```

Comment:
Should this be added to the transport request time object rather than the store result?

Reply:
IMO, no, because transport request time object only cares about SDK timelines for all the events. It specifies the time spent in every step from the request being created to sent and response received. 
This needs to be part of store Result, as we are getting this latency from the backend, hence making it very clear about the time backend took to process this result.